### PR TITLE
Turning in Mini Project 3

### PR DESCRIPTION
**Pull request by segerpeter07**
Website link in the description

### CODE REVIEW

**Comment by SungwooPark** [comment]
I can't seem to find a website link. I am assuming this website was your project writeup?

File: data_grabber.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import twitter
```
Comment (SungwooPark) [inline data_grabber.py:1]: It is a good idea to include a header comment in all files that you wrote for your program.

**Comment by segerpeter07** [comment]
Sorry about that! Here's the link

http://peterhenryseger.com/TwitterFeels?


________________________________

File: data_grabber.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    file.write(items.text + "+++")
+
+# get statuses 2
+statuses = api.GetUserTimeline(screen_name=user2, count=200)
+file = open('data2.txt', 'w')
+for items in statuses:
+    file.write(items.text + "+++")
+
+
+text_analyzer.main(user1, user2)
```
Comment (SungwooPark) [inline data_grabber.py:33]: Nice work separating your data fetching functionality from your analysis into two separate files.

**Comment by SungwooPark** [comment]
Nice work. Your implementation was very clear and readable. Nice amount of documentation to help readers understand your code. Just include a link to your website somewhere in your `README.md` when you have time.

**Pull request closed without merging**